packages/model-viewer/src/test/features/scene-graph/nodes/primitive-node-spec.ts

```diff
@@ -170,6 +171,79 @@ suite('scene-graph/model/mesh-primitives', () => {
     });
   });
 
+  suite('Mesh with multiple primitives each with variants', () => {
+    let model: Model;
+    setup(async () => {
+      const threeGLTF = await loadThreeGLTF(MESH_PRIMITIVES_GLB_PATH);
+      model = new Model(CorrelatedSceneGraph.from(threeGLTF));
+    });
+
+    test('Primitive count matches glTF file', async () => {
+      expect(model![$primitivesList].length).to.equal(3) ;
+    });
+
+    test('Primitives should have expected variant names', async () => {
+      expect(findPrimitivesWithVariant(model, 'Normal')).to.not.be.null;
+      expect(findPrimitivesWithVariant(model, 'Inverse')).to.not.be.null;
+    });
+
+    test('Switching to incorrect variant name', async () => {
+      const primitive = findPrimitivesWithVariant(model, 'Normal')![0];
+      const material = await primitive.enableVariant('Does not exist');
+      expect(material).to.be.null;
+    });
+
+    test('Switching to current variant', async () => {
+      const primitives = findPrimitivesWithVariant(model, 'Normal')!;
+      const materials = new Array<MeshStandardMaterial>();
+      for (const primitive of primitives) {
+        materials.push(
+            await primitive.enableVariant('Inverse') as
+                MeshStandardMaterial);
+      }
+      expect(materials).to.not.be.empty;
+      expect(materials.find((material: MeshStandardMaterial) => {
+        return material.name === 'STEEL BLACK';
+      })).to.not.be.ok;
```

Comment:
Can we check that it does equal something instead of doesn't, to make it more specific?